circle.yml

```diff
@@ -12,9 +12,10 @@ machine:
     testpilot.dev: 127.0.0.1
   services:
     - docker
-
   node:
     version: 6.2.0
+  environment:
+    TESTPILOT_STATIC_BRANCHES: development-static 1305-revamped-static
```

Comment:
And just updated the commit to remove this, since I want to merge soon and this doesn't belong in the actual project